source/extensions/grpc_credentials/aws_iam/config.cc

```diff
@@ -39,7 +39,8 @@ std::shared_ptr<grpc::ChannelCredentials> AwsIamGrpcCredentialsFactory::getChann
                 credential.from_plugin(), ProtobufMessage::getNullValidationVisitor(),
                 credentials_factory);
         const auto& config = Envoy::MessageUtil::downcastAndValidate<
-            const envoy::config::grpc_credential::v2alpha::AwsIamConfig&>(*config_message);
+            const envoy::config::grpc_credential::v2alpha::AwsIamConfig&>(
+            *config_message, ProtobufMessage::getNullValidationVisitor());
```

Comment:
Not really related to this change per se, but isn't this a bug in that we aren't doing all the normal stuff for gRPC plugins? Should we track this via TODO, etc.?

Reply:
Fair point, tracking at https://github.com/envoyproxy/envoy/issues/8010.